python-client/pyproject.toml

```diff
@@ -162,7 +162,7 @@ llm = [
     "evaluate",
     #    pdm lock -G:all doesn't work without fixing these two versions
     "datasets>=2.13.0",
-    "bert-score>=0.3.13",
```

Comment:
Have we removed completely BERT score? Isn’t it used in other detectors?

Reply:
bert not in other detectors